critiquebrainz/db/oauth_client.py

```diff
@@ -0,0 +1,137 @@
+from critiquebrainz import db
+from critiquebrainz.db import exceptions as db_exceptions
+from critiquebrainz.utils import generate_string
+import sqlalchemy
+
+
+def create(*, user_id, name, desc, website, redirect_uri):
+    """Create new OAuth client and generates secret key for it.
```

Comment:
Should be either "Create**s** new OAuth client and generates a secret key for it." or "Create new OAuth client and generate a secret key for it."